src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -744,7 +744,7 @@ public class OpcodeSetMember : Opcode
         public override void Execute(ICpu cpu)
         {
             Structure value = cpu.PopStructureEncapsulated();         // new value to set it to
-            string suffixName = cpu.PopStack().ToString().ToUpper();  // name of suffix being set
+            string suffixName = cpu.PopStack().ToString();            // name of suffix being set
```

Comment:
Although, there's a part of me that wants to ILSpy on the contents of the actual string compare and see what it does when you tell it to use ``OrdinalIgnoreCase``.  I have a nagging suspicion it may end up generating a temp case converted string in its head anyway, which would actually be more GC usage then if it's happening on every comparison.